coda-automation/terraform/modules/elasticsearch/main.tf

```diff
@@ -0,0 +1,79 @@
+locals {
```

Comment:
same here (re: migrating old/obsolete configs) -- definitely could reduce a bit of the noise when trying to understand the src/infra